Utils.py

```diff
@@ -37,7 +37,7 @@ class Version(typing.NamedTuple):
     build: int
 
 
-__version__ = "0.3.5"
+__version__ = "0.3.6"
```

Comment:
this supposed to be here?

Reply:
"It also updates the required client version and AP version to 0.3.6"
It kind of has to be for the client version check

Reply:
Which, I made no changes to the client and it works on IDs, so it should work, but I'd rather people update.